static/downloadables/grep_file.py

```diff
@@ -116,8 +140,45 @@ def formatVarForFiltering(traits, studyTypes, studyIDs, ethnicities, sexes, valu
 
     return traits, studyTypes, studyIDs, ethnicities, sexes, valueTypes
 
-
-def filterTXT(tableObjDict, clumpsObjDict, studySnpsDict, inputFilePath, filteredFilePath, traits, studyIDs, studyTypes, ethnicities, valueTypes, sexes, isAllFiltersNone, p_cutOff):
+def getPreferredPop(popList, superPop):
```

Comment:
I would just keep this in the connect_to_server.py, put `from connect_to_server import getPreferredPop`  up at the top and use that to call this function. Just to keep from repeating code.